packages/prisma/schema.prisma

```diff
@@ -242,7 +242,7 @@ model User {
 }
 
 model Team {
-  id                  Int                @id @default(autoincrement())
+  id                  Int                     @id @default(autoincrement())
```

Comment:
can you uncommit your schema.prisma changes?

Reply:
and uncommit yarn.lock

Reply:
Okk sure !!

Reply:
I removed them , review it please